routes/era5wrf.py

```diff
@@ -0,0 +1,559 @@
+from . import routes
+
+era5wrf_api = Blueprint("era5wrf_api", __name__)
+
+era5wrf_coverage_ids = {
+    "t2_min": "era5_4km_daily_t2_min",
+    "t2_mean": "era5_4km_daily_t2_mean",
+    "t2_max": "era5_4km_daily_t2_max",
+    "rh2_min": "era5_4km_daily_rh2_min",
+    "rh2_mean": "era5_4km_daily_rh2_mean",
+    "rh2_max": "era5_4km_daily_rh2_max",
+    "wspd10_max": "era5_4km_daily_wspd10_max",
+    "wspd10_mean": "era5_4km_daily_wspd10_mean",
+    "wdir10_mean": "era5_4km_daily_wdir10_mean",
+    "seaice_max": "era5_4km_daily_seaice_max",
+    "rainnc_sum": "era5_4km_daily_rainnc_sum",
+}
+
+era5wrf_meta = {
+    key: asyncio.run(describe_via_wcps(cov_id))
+    for key, cov_id in era5wrf_coverage_ids.items()
+}
+
+
+def format_era5_datetime(date_obj):
+    """Convert datetime object to ISO format.
+
+    CP note: good candidate to move to a utility module for time slicing and time validation
+
+    Args:
+        date_obj (datetime): Python datetime object
+
+    Returns:
+        str: ISO formatted datetime string
+    """
+    if date_obj is None:
+        return None
+    return f"{date_obj.strftime('%Y-%m-%d')}T00:00:00.000Z"
+
+
+def create_era5_time_slice(start_date, end_date):
+    """Create time_slice tuple for ERA5-WRF WCS requests.
+
+    CP note: good candidate to move to a utility module for time slicing and time validation
+    This is a small function and could be bundled but it helps me debug
+
+    Args:
+        start_date (datetime): Start date for temporal slice
+        end_date (datetime): End date for temporal slice
+
+    Returns:
+        tuple: (time_axis, time_range_string) for WCS request, or None if no dates
+    """
+    if start_date is None or end_date is None:
+        return None
+
+    start_iso = format_era5_datetime(start_date)
+    end_iso = format_era5_datetime(end_date)
+    return ("time", f'"{start_iso}","{end_iso}"')
+
+
+def get_era5_temporal_bounds(coverage_meta):
+    """Extract temporal bounds from ERA5-WRF coverage metadata.
+
+    Args:
+        coverage_meta (dict): Coverage metadata from describe_via_wcps()
+
+    Returns:
+        tuple: (min_date, max_date) as datetime objects
+    """
+    try:
+        # Find the time axis in coverage metadata
+        # CP note: good candidate to move to a function in validate_request
+        # much like the function that determines the spatial axes
+        time_axis = next(
+            axis
+            for axis in coverage_meta["domainSet"]["generalGrid"]["axis"]
+            if axis["axisLabel"]
+            == "time"  # ERA5-WRF uses "time" axis, could also be "ansi"
+        )
+
+        # Extract coordinate list and get bounds
+        time_coordinates = time_axis["coordinate"]
+
+        # Convert to datetime objects
+        min_date = pd.to_datetime(time_coordinates[0])
+        max_date = pd.to_datetime(time_coordinates[-1])
+
+        # Remove timezone info to make naive datetimes
+        min_date_naive = (
```

Comment:
Possible concern here, what's the time zone for the incoming ERA5 data?  Why would we shift these to be naive?